### PR TITLE
added something special

### PR DESCRIPTION
**Pull request by gruzzlymug**

### CODE REVIEW

File: app/models/blah.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Blah < ActiveRecord::Base
```
Comment (FreeBreadsticks) [inline app/models/blah.rb:1]: why did you name it this

File: test/models/blah_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class BlahTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (FreeBreadsticks) [inline test/models/blah_test.rb:4]: the truth is out there

File: app/models/blah.rb (hunk 1 above)
Comment (gruzzlymug) [inline app/models/blah.rb:1]: it's the best i could do in a pinch

File: test/models/blah_test.rb (hunk 2 above)
Comment (gruzzlymug) [inline test/models/blah_test.rb:4]:
```
<O>
```

Comment (FreeBreadsticks) [inline test/models/blah_test.rb:4]:
```
a=6
```

this is code

**Comment by FreeBreadsticks** [comment]
👍

**Pull request merged**